kernel/README.md

```diff
@@ -1,13 +1,13 @@
 # Build Kata Containers Kernel
 
-This document explains the steps to build a compatible kernel with Kata
-Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers. 
+This document explains the steps to build a kernel recommended for use with
+Kata Containers Containers. To do this use `build-kernel.sh`, this script
```

Comment:
two Containers intentionally？

Reply:
Good catch @caoruidong. It can make sense with two but I agree that we could just drop one of them :smile:

Maybe "*I Feel For You*" by Chaka Khan was playing when this was written. Sorry, bad joke :smile:

Reply:
/me lloks for "I Feel For You" and fix and drops one `Contaienrs`